administration/src/application/components/forms/WorkAtDepartmentEntitlementForm.tsx

```diff
@@ -29,21 +29,23 @@ const WorkAtDepartmentEntitlementForm: Form<State, Options, ValidatedInput, Addi
   Component: ({ state, setState }) => (
     <>
       <CustomDivider label='Angaben zur Tätigkeit' />
-      <OrganizationForm.Component
+      <SubForms.organization.Component
         state={state.organization}
         setState={useUpdateStateCallback(setState, 'organization')}
       />
-      <ShortTextForm.Component
+      <SubForms.responsibility.Component
         label='Funktion oder Tätigkeit'
         state={state.responsibility}
         setState={useUpdateStateCallback(setState, 'responsibility')}
       />
       <h4>Tätigkeitsnachweis</h4>
       <p>
-        Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. Die Datei darf maximal{' '}
-        {FILE_SIZE_LIMIT_MEGA_BYTES} MB groß sein und muss im JPG, PNG oder PDF Format sein.
+        Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. {FileRequirementsText}
```

Comment:
Should this text not be adjusted to 
```suggestion
        Falls vorhanden: hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. {FileRequirementsText}
```

Reply:
Here, I agree, it seems I forgot to change it here. (And also make the certificate optional)

In general, I think we should only make the file upload optional for the cases where we contact the corresponding organization via email (#627) - this should be the case whenever the user can specify an organization & corresponding contact person.